linker/packages/linker/src/GleamImport.ts

```diff
@@ -29,7 +29,7 @@ import {
 import { digits, eol, word } from "./MatchWgslD.js";
 import { makeElem } from "./ParseSupport.js";
 
-const gleamImportSymbolSet = "/ { } , ( ) .. . * ;";
+const gleamImportSymbolSet = "/ { } , ( ) .. . * ; @ #"; // Had to add @ and # here to get the parsing tests to work. Weird.
```

Comment:
We could explore the mystery, but presumably this gets revisited for new import syntax anyway, and nice work keeping tests alive while we do!